etc/ProdOfflineConfiguration.py

```diff
@@ -1032,15 +1032,16 @@
 for dataset in datasets:
     addDataset(tier0Config, dataset,
                do_reco = True,
-               write_reco = False, write_aod = True, write_miniaod = False, write_dqm = False,
+               write_reco = False, write_aod = True, write_miniaod = False, write_dqm = True,
                global_tag = hiPromptrecoGlobalTag,
+               dqm_sequences = [ "@common" ],
```

Comment:
Please remove this line. The @common sequence is a smallish subset of all DQM sequences, I am not sure its enough here. Just don't configure any dqm_sequences for the HI datasets, this will use the defaults then.